src/views/Project/index.jsx

```diff
@@ -42,6 +43,9 @@ const tagReposMapping = repositories =>
       ...mappings,
     };
   }, {});
+const pagingGithub = {};
+const pagingGood = {};
+const pagingMen = {};
```

Comment:
@helfi92 any advise on where to track all the paging cursor variables?

Reply:
Maybe something like

```jsx
const pageCursors = {
  github: {},
  bzGoodFirst: {},
  bzMentored: {},
};
```